src/extensions/main-api/k8s-api.ts

```diff
@@ -3,7 +3,14 @@
  * Licensed under MIT License. See LICENSE in root directory for more information.
  */
 
-export { isAllowedResource } from "../../common/utils/allowed-resource";
+/**
+ * @deprecated This function never works
```

Comment:
Is this really a good idea in a minor version?

Reply:
The old version would always return `false` because on main a call to `getHostedClusterId` always returns undefined.